### PR TITLE
DHFPROD-10240: check e2e win-chrome failures in the latest chrome version

### PR DESCRIPTION
**Pull request by amejiaq**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran newly added/edited cypress tests on Firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [n/a] Added to Release Wiki

### CODE REVIEW

**Review by aniamy — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@amejiaq I see lot of explicit waits have been added as a part of this PR, Did you look into any alternate solutions before we decided on this option ?

**Comment by amejiaq** [comment]
> @amejiaq I see lot of explicit waits have been added as a part of this PR, Did you look into any alternate solutions before we decided on this option ?

That is the way I believe fixes the issue, some time the test run so fast and the app doesn't have enough time to process that is that I thought

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/run/runFlow.cy.tsx [hunk 1]
```diff
@@ -274,6 +274,8 @@ describe("Run Tile tests", () => {
     runPage.deleteStep("map-orders", flowName).click();
     runPage.deleteStepConfirmationMessage("map-orders", flowName);
     cy.findByLabelText("Yes").click();
+    cy.waitForAsyncRequest();
+    cy.wait(5000);
 
     cy.intercept("GET", "/api/jobs/**").as("runResponse");
     runPage.runFlow(flowName);
```
Comment (ngodugu-marklogic) [inline marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/run/runFlow.cy.tsx:281]: @amejiaq Can we add explicit wait instead and try it out. In this case, in runPage.runFlow(flowName) method we can add cy.get(`#runFlow-${flowName}`).should("be.visible", {timeout: 2000}).click({force: true});

Comment (amejiaq) [reply marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/run/runFlow.cy.tsx]: fixed 🛠

**Review by vmendez-marklogic — DISMISSED** [review]

**Comment by vmendez-marklogic** [comment]
Changes looks good, but it has one cypress failure that needs to be check

**Review by ngodugu-marklogic — DISMISSED** [review]

**Review by littlecherro — APPROVED** [review]

**Review by ngodugu-marklogic — APPROVED** [review]

**Pull request merged**